File: swagger/models/get_staff_appointments_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetStaffAppointmentsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used. ,

		appointments
		Contains information about appointments and their details.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Appointments': 'appointments',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'appointments': 'Appointments',
		}

File: swagger/Models/GetStaffAppointmentsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetStaffAppointmentsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used. ,
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about appointments and their details.
	 * 
	 * @var Appointment[] 
	 */
	public $appointments;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Appointments' => ['appointments', 'array', Appointment::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'appointments' => ['Appointments', 'array', Appointment::class],
			];
	}
}
